### DISCUSSION TITLE
oap and ui server always restart, how could i resolve it ?

### DISCUSSION THREAD

**Discussion by houbinglei** [apache/skywalking#11426] [Q&A]
version: 9.5.0
first , i try binary deployment with oap/startup.sh; then i replace this way with skywalking-helm deploy.
but the java server with oap and ui with binary deploied always restart , Even though I do `kill -9 `, it will always restart.
how can i stop the oap and ui server deploied with oap/startup.sh?

**Comment by wu-sheng** [comment]
You need to find the reason. It is impossible for us to know.

**Reply by houbinglei** [reply]
oh! thanks! I'm sorry to bring this up...
i started a container, and i forgot it ....
i hope this will help others if have the same question....
